lib/ui/window.dart

```diff
@@ -714,6 +714,15 @@ class Window {
   /// semantics update cannot be used further.
   void updateSemantics(SemanticsUpdate update) native 'Window_updateSemantics';
 
+  /// Set the debug name associated with this window's isolate.
```

Comment:
Clarify that this is referring to the root isolate for the window and not any of its children.